EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -7,6 +7,7 @@ public struct MediaPlayerView: View {
     @ObservedObject var viewModel: MediaPlayerView.ViewModel
     @State private var showActionButtons = false
     @State private var screenWidth: CGFloat = .zero
+    @State private var showSeek: Seek?
```

Comment:
this variable looks more like a `visibleSeek. `showSeek` is what you would name a boolean